src/css.js

```diff
@@ -28,6 +28,7 @@ var
 	// except "table", "table-cell", or "table-caption"
 	// See here for display values: https://developer.mozilla.org/en-US/docs/CSS/display
 	rdisplayswap = /^(none|table(?!-c[ea]).+)/,
+	rnonemptywhite = /^\s+$/,
```

Comment:
And even if it weren't for what I just wrote just rnotwhite would not be
enough as we need to catch strings that consist only of whitespaces but
which are not empty.

On Wednesday, 7 September 2016, Richard Gibson <notifications@github.com>
wrote:

> In src/css.js
> <https://github.com/jquery/jquery/pull/3308#discussion_r77842189>:
>
> > @@ -28,6 +28,7 @@ var
> >  	// except "table", "table-cell", or "table-caption"
> >  	// See here for display values: https://developer.mozilla.org/en-US/docs/CSS/display
> >  	rdisplayswap = /^(none|table(?!-c[ea]).+)/,
> > +	rnonemptywhite = /^\s+$/,
>
> Per gh-3072 <https://github.com/jquery/jquery/issues/3072>, \s is too
> broad. Rather than introducing a new regex, we should just use rnotwhite
> and let @timmywil <https://github.com/timmywil>'s updates thereof perform
> the final step of this fix.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/jquery/jquery/pull/3308/files/5576d152b11b8aaf3ebafbb5ba1b64caf7a8e261#r77842189>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABrUnoKGINa-0iIgnYnWt1jtnXI6o0d2ks5qntQ4gaJpZM4Jxm7Y>
> .
>


-- 
Michał Gołębiowski